### PR TITLE
blocksmc bypasses

### PR DESCRIPTION
**Pull request by liquidsquid1**
added disablers useful for going more blatant, and added a speed mode which is good for hvhs.

### CODE REVIEW

**Comment by liquidsquid1** [comment]
detekt 🖕

**Comment by 1zun4** [comment]
Can we stop calling everything a disabler? Those two "Disablers" are just bypasses for one-specific module.

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerBlocksMCVelocity.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+) {
+
+    val repeatable = repeatable {
+        when (player.hurtTime) {
+            9 -> {
+                // two in the same tick so you dont flag movement
+                network.sendPacket(ClientCommandC2SPacket(player, ClientCommandC2SPacket.Mode.PRESS_SHIFT_KEY))
+                network.sendPacket(ClientCommandC2SPacket(player, ClientCommandC2SPacket.Mode.RELEASE_SHIFT_KEY))
+            }
+        }
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerBlocksMCVelocity.kt]: Not a disabler.

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerBlocksMCScaffold.kt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            return@repeatable
+        }
+
+        if (player.isOnGround) {
+            player.velocity = player.velocity.multiply(
+                0.99,
+                1.0,
+                0.99
+            )
+        }
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerBlocksMCScaffold.kt]: Not a disabler.

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/blocksmc/SpeedBlocksMC.kt [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+import net.ccbluex.liquidbounce.event.repeatable
+import net.ccbluex.liquidbounce.utils.entity.moving
+import net.ccbluex.liquidbounce.utils.entity.sqrtSpeed
+import net.ccbluex.liquidbounce.utils.entity.strafe
+import net.minecraft.entity.effect.StatusEffects
+
+/*
+    extensive blocksmc speed
+    @author liquidsquid1
+ */
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/blocksmc/SpeedBlocksMC.kt]: Format comment

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/blocksmc/SpeedBlocksMC.kt [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        canSpeed = false
+    }
+
+    override fun disable() {
+        airTicks = 0
+        player.strafe(speed = 0.0)
+    }
+
+    val repeatable = repeatable {
+
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/blocksmc/SpeedBlocksMC.kt]: space?

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerBlocksMCVelocity.kt (hunk 1 above)
Comment (liquidsquid1) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerBlocksMCVelocity.kt]: it disables the verus velocity checks so you can use whatever velocity you want ?

**Comment by liquidsquid1** [comment]
> Can we stop calling everything a disabler? Those two "Disablers" are just bypasses for one-specific module.

i'll see how i can resolve this

**Pull request merged**